examples/BUILD

```diff
@@ -20,24 +20,31 @@ licenses(["notice"])  # Apache 2.0
 bazel_integration_test(
     name = "pip_install_example",
     timeout = "long",
+    tags = ["integration-test"],
```

Comment:
nit: why doesn't `bazel_integration_test` add the tags? it's already a macro

Reply:
Right, can do that.

Reply:
Fixed in https://github.com/bazelbuild/rules_python/pull/855/commits/913daf82a4503e21038d6b235e2e29af76cdd7b2.